### ISSUE TITLE
picking nits :)

### ISSUE THREAD

**Issue by quepol** [18F/cg-landing#16]
Howdy, all. The migration to the SBST template looks lovely! Nice job. Here are a few of the things I was mentioning in Slack that I think could clean it up even more:
- [x] re ^^ make sure the background is dark enough for a [4.5:1 contrast ratio](http://webaim.org/resources/contrastchecker/)
- [x] remove the icons from the About section -- will make it cleaner, and they don't really relate
- [x] after ^^ you could bump up the font size of those headings 5-10%
- [x] need to modify the hero area for the mobile breakpoints (title text goes outside background box)
- [x] need to modify the header/nav for mobile breakpoints (fails between 600px-840px)
- [x] lowercase "g" on government in 2nd blurb title
- [x] "focus on" instead of "focus in" in 3rd blurb

Totally personal preference stuff:
- [x] vertically center the hero text on the cloud image (looks better w/ this design; also looks ok at the top)
- [x] "Your team, unleashed." instead of all caps
- [x] ^^ remove the sans serif font and just let it default to the serif
- [x] "Sign up to receive updates..." at `font-size: 400;` instead of 700

$0.02

**Comment by mogul** [comment]
Hi @quepol, I took care of most of these. Regarding this one:

> - [ ] re ^^ make sure the background is dark enough for a [4.5:1 contrast ratio](http://webaim.org/resources/contrastchecker/)

The checker's not turning up a problem.... Tried #ffffff vs #212121 and #0d0d0d (x21 at 0.4 alpha). Are you just eyeballing it or do you know for sure there's a problem?

Also on these:

> - [ ] need to modify the hero area for the mobile breakpoints (title text goes outside background box)
> - [ ] need to modify the header/nav for mobile breakpoints (fails between 600px-840px)

Do you have any pointers about how to replicate those/where I would change them? I'm completely unfamiliar with that.

**Comment by maya** [comment]
@quepol @mogul I tested the lightest part I could find on that image against white text and it passed WCAG AA for large text, but not small text. Since this is a large heading, maybe that's OK?

**Comment by maya** [comment]
@quepol @mogul I fixed the mobile issues you pointed out in my latest commits.

**Comment by mogul** [comment]
Thanks @maya for the spot help, and @quepol for helping us keep our standards high. :)
